docs/clinic-login.txt

```diff
@@ -0,0 +1,13 @@
+
+  <title>Clinic Login</title>
+
+  Log in to the Clinic Upload server.
+
+  <h1>Flags</h1>
+  -h | --help                Display Help
+  --upload-url               Specify a custom URL to a Clinic Upload server.
```

Comment:
Maybe we should rename this option to something else? `--server` maybe?

Reply:
agree